src/webportal/src/app/job-submission/components/tab-form-content.jsx

```diff
@@ -39,6 +39,8 @@ import { getDefaultContainerSize } from '../models/container-size';
 import { PROTOCOL_TOOLTIPS } from '../utils/constants';
 import config from '../../config/webportal.config';
 
+const TASK_ROLE_NAME_REGX = /^[a-z][a-z0-9]*$/;
```

Comment:
Please refer https://github.com/microsoft/frameworkcontroller/blob/d602cdcbab2b91a7772f9c0d956d17e5f9d69847/pkg/apis/frameworkcontroller/v1/crd.go#L33 for task name